ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -1142,7 +1144,7 @@ func (pg *DEXOnboarding) connectServerAndPrepareForBonding() {
 		xc, err = dexClient.GetDEXConfig(pg.bondServer.url, pg.bondServer.cert)
 	}
 	if err != nil {
-		pg.notifyError(fmt.Errorf("Error retrieving DEX server info: %w", err).Error())
+		pg.notifyError(fmt.Sprintf("Error retrieving DEX server info: %s", err.Error()))
```

Comment:
Was this change necessary?

Reply:
because we don't return error, so why we need create error for it, additionally, it will cause a warning when the returned message is capitalized